cmd/access_test.go

```diff
@@ -0,0 +1,13 @@
+package cmd
+
+import (
+	"testing"
+)
+
+func TestAccessCreateToken(t *testing.T) {
+	run := NewTestRunner(t, "peer", "cmd_test_create_access_token")
+	defer run.Delete()
+
+	run.MustExecCombinedOutErr(t, "qri access token --for me")
```

Comment:
These aren't using the result of stdout or stderr. Just use `run.MustExec` instead.